image-loader/app/controllers/Application.scala

```diff
@@ -106,20 +108,22 @@ class ImageLoader extends Controller with ArgoHelpers {
       mimeType = mimeType_,
       uploadTime = uploadTime_,
       uploadedBy = uploadedBy_,
-      identifiers = identifiers_
+      identifiers = identifiers_,
+      uploadInfo = uploadInfo_
     )
 
     Logger.info(s"Received ${uploadRequestDescription(uploadRequest)}")
 
-    val supportedMimeType = Config.supportedMimeTypes.exists(Some(_) == mimeType_)
+    val supportedMimeType = Config.supportedMimeTypes.contains(mimeType_)
```

Comment:
Does that work? I thought `mimeType_` was an `Option[String]` and `Config.supportedMimeTypes` a `List[String]`.